tests/src/whisk/core/cli/test/WskEntitlementTests.scala

```diff
@@ -29,37 +31,77 @@ import common.TestUtils.TIMEOUT
 import common.Wsk
 import common.WskProps
 import common.WskTestHelpers
-import spray.json.DefaultJsonProtocol._
+import spray.json.DefaultJsonProtocol.StringJsonFormat
```

Comment:
You can tell it to leave `_` imports alone. I find it more convenient to just import everything for spry.json, so everything "just works".